src/payment/coupon/CouponForm.jsx

```diff
@@ -43,7 +41,7 @@ export class CouponForm extends Component {
     switch (errorCode) {
       // Cases that need a `code`
       case 'empty_basket':
-      case 'already_applied_voucher':
+      case 'already_applied_coupon':
```

Comment:
Has this code been changed on the backend as well?

Reply:
Good catch, changed back to `already_applied_voucher`